Updates pricing plans features and dashboard links

makeavish opened the pull request:
Adds new features to community tier including:
- OTel-native messaging queue monitoring
- Signal correlation
- Large trace visualization
- Unlimited dashboards

Updates hostmetrics dashboard JSON file references

Calm-Rock commented on docs/logs-pipelines/guides/json.md:
```diff
@@ -0,0 +1,267 @@
+  - Select `Json Parser` in the **Select Processor Type** field.
+  - Use the **Name of Json Parser Processor** field to set a short descriptive name for the processor.
+  - Set the **Parse From** field to `body` <br/>
+  - Use **Parse To** field to define the attribute where the parsed JSON body should be stored temporarily. For example `attributes.temp_parsed_body`.
```
```suggestion
  - Use **Parse To** field to define the attribute where the parsed JSON body should be stored temporarily. For example `attributes.temp_parsed_body`.

<br> </br>
```

Add a break line between the last pointer and the image to create some spacing.

Calm-Rock commented on docs/logs-pipelines/guides/json.md:
```diff
@@ -0,0 +1,267 @@
+  - Use the **Name of Move Processor** field to set a short descriptive name for the processor.
+  - Set the **From** field to the path of the parsed JSON field to be extracted. For example `attributes.temp_parsed_body.country`
+  - Use **To** field to define the attribute where the JSON field should be stored.
+<br/>
```
```suggestion

<br> </br>

```
the `<br/>` didn't worked I guess. It might be an issue with docusauraus.

Calm-Rock commented on docs/logs-pipelines/guides/json.md:
```diff
@@ -0,0 +1,267 @@
+  - Select `Remove` in the **Select Processor Type** field.
+  - Use the **Name of Remove Processor** field to set a short descriptive name for the processor.
+  - Set **Field** input to path of the attribute we used for storing parsed JSON body temporarily. For example `attributes.temp_parsed_body`
+<br/>
```
```suggestion

<br> </br>

```
the <br/> didn't worked here as well

Calm-Rock commented on docs/logs-pipelines/guides/json.md:
```diff
@@ -0,0 +1,267 @@
+
+## Step 5: Save Pipelines and Verify
+
+Once you have previewed your pipeline and verified that it will work as expected, press the **Save Configuration** button at the bottom of the pipelines list to save pipelines. This will store the latest state of your pipelines and will deploy them for pre-processing.
```
You can add a relevant image for this step as well

Calm-Rock commented on docs/logs-pipelines/guides/trace.md:
```diff
@@ -0,0 +1,294 @@
+  </figcaption>
+</figure>
+<br/>
+
```
It would be good if it is clarified what the user will learn in this guide.

Something like what is mentioned in the JSON Logs guide : "In this guide, you will see how to parse interesting fields out of serialized JSON bodies into their own log attributes."

Calm-Rock commented on docs/logs-pipelines/guides/trace.md:
```diff
@@ -0,0 +1,294 @@
+</figure>
+<br/>
+
+Wait for a few minutes to let the pipelines deploy and for the latest batches of logs to get pre-processed and stored in the database. Then you can head over to the logs explorer to verify that your logs are getting pre-processed as expected.  
```
No. Let's not include it. 
Since you have already mentioned what the result will be, it seems good.

Calm-Rock commented:
LGTM. Merging it!

makeavish committed:
Updates pricing plans features and dashboard links

Adds new features to community tier including:
- OTel-native messaging queue monitoring
- Signal correlation
- Large trace visualization
- Unlimited dashboards

Updates hostmetrics dashboard JSON file references

makeavish committed:
fix: clarify SAML authentication availability in documentation
